server/server.py

```diff
@@ -55,6 +77,22 @@ def submit():
     return resp
 
 
+@app.route(
+    "/run",
+    methods=["POST"],
+)
+def run():
+    test_location = os.getenv("TESTS_DIR")
+    os.chdir(test_location)
+    command = "pipenv run pytest"
+    subprocess.Popen(
+        command, shell=True, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT
+    )
+    resp = jsonify("Success")
```

Comment:
JSON can be tricky, but do we need to display the results of STDOUT here at all?
It looks like we're throwing captured STDOUT into The Void and redirecting STDERR to STDOUT, but not sure we're logging it anywhere or if we need to.

Reply:
Hmm yeah, I know I'll need it for debugging but it's not hard to add back.